### ISSUE TITLE
ICS Parser has changed ownership

### ISSUE THREAD

**Issue by u01jmg3** [unclecheese/silverstripe-event-calendar#115]
Hello,

ICS Parser has changed ownership so I recommend updating your [`composer.json`](https://github.com/unclecheese/silverstripe-event-calendar/blob/master/composer.json#L17) to reflect this.

```json
"u01jmg3/ics-parser": "1.0.4"
```

**Comment by vixen1975** [comment]
Yes, I just tried to download this module via Composer and it failed as it can not find the ics-parser module as a requirement.

**Comment by u01jmg3** [comment]
My apologies - in order to maintain the statistics already accumulated by the library I had to continue using ICS Parser under the previous owner with myself added as a maintainer on Packagist. You therefore have to do nothing. I would only suggest updating to `2.0.0` when you are ready.

**Comment by mrbookt** [comment]
@vixen1975 add this line to the composer.json before event calendar:

`johngrogg/ics-parser": "1.0.4 as 1.0.3"`
